### PR TITLE
Feature/tech details endpoint

### PR DESCRIPTION
**Pull request by deadbert**
## Description

Feature added: Technology details endpoint
- Spec tests done
- Facade and Service Methods added for new feature
- New Poro and Serializer for the new format of data

To-Do:
- currently this is all very Happy path, and expects Front End app to make proper requests, we should move onto making some error handling and sad pathing next

## Type of change

- [ ] fix
- [x] feat
- [x] test
- [ ] refactor
- [ ] docs

## Checklist

- [x] code has been self reviewed
- [x] code runs without any errors
- [x] thorough testing has been implemented if adding feature
- [x] all tests pass

### Thanks!

### CODE REVIEW

File: spec/api/v1/requests_spec.rb [hunk 1]
```diff
@@ -61,4 +61,56 @@
...
+      expect(result[:data][:attributes]).to have_key :compatibility
+      expect(result[:data][:attributes]).to have_key :versatility
+      expect(result[:data][:attributes]).to have_key :professional_features
+      expect(result[:data][:attributes]).to have_key :reliability
+      expect(result[:data][:attributes]).to have_key :performance
+      expect(result[:data][:attributes]).to have_key :community_engagement
+      expect(result[:data][:attributes]).to have_key :detailed_description
+    end
+  end
 end
```
Comment (jeremiahblackol) [inline spec/api/v1/requests_spec.rb:116]: Good job getting this tested, when you add sad paths/error handling please tag me in that PR too. Conceptually sad paths make sense to me across stacks - would love to see how they're written though!

**Review by jeremiahblackol — COMMENTED** [review]
Good job w/ documentation.

File: README.md [hunk 2]
```diff
@@ -53,6 +53,13 @@ example request: <br> ![Picture of example request.](example_request.png)<br>
 - returns assistive technologies that the AI finds for the given tech needs and disability description <br>
 example response: <br> ![Picture of response.](example_response.png) <br>
 
+### Get Endpoints
+#### "/api/v1/ai_requests" <br>
+example request: <br> ![Picture of example request.](example_tech_details_request.png)<br>
```
Comment (elizakeating) [inline README.md:58]: Nice pictures!

File: spec/api/v1/requests_spec.rb [hunk 3]
```diff
@@ -61,4 +61,56 @@
...
+
+      expect(result).to have_key :data
+      expect(result[:data]).to be_a Hash 
+      expect(result[:data]).to have_key :id
+      expect(result[:data]).to have_key :type
+      expect(result[:data]).to have_key :attributes
+
+      expect(result[:data][:type]).to eq("detailed_tech")
+      expect(result[:data][:attributes]).to have_key :compatibility
+      expect(result[:data][:attributes]).to have_key :versatility
```
Comment (elizakeating) [inline spec/api/v1/requests_spec.rb:108]: Nice testing!

**Review by elizakeating — APPROVED** [review]
This looks really solid. Will be good to implement!

**Pull request merged**